requirements.txt

```diff
@@ -16,3 +16,4 @@ python-bidi
 IPython
 Jupyter
 PyQt5
+sklearn
```

Comment:
we should add this requirement to the setup.py file too

Reply:
Requirement added.